src/Tribe/Views/V2/Views/Month_View.php

```diff
@@ -346,12 +346,13 @@ static function ( $event ) use ( $date_object ) {
 			);
 
 			$start_of_week = get_option( 'start_of_week', 0 );
+			$is_start_of_week = (int)$start_of_week === (int)$date_object->format( 'w' );
```

Comment:
* :no_entry_sign: Spaces must be used to indent lines; tabs are not allowed
* :no_entry_sign: Expected 1 space(s) after cast statement; 0 found
* :no_entry_sign: Expected 1 space(s) after cast statement; 0 found
* :warning: Line exceeds 80 characters; contains 89 characters

<sub>hash: eff4a4616f01a51e404f13a4f73254d9</sub>